src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -0,0 +1,341 @@
+          </paper-card>  
+        </outline-step-view>
+        
+        <outline-step-view id="projectSetup" display-action="">
+          <span slot="step-title">Create your Google Cloud Platform project.</span>
+          <span slot="step-description">This will create a new project on your GCP account to hold your Outline servers.</span>
+          <span slot="step-action">
+            <paper-button 
+                id="createServerButton" 
+                @tap="${this.handleProjectSetupNextTap}" 
+                ?disabled="${
+        !this.isProjectSetupNextEnabled(this.selectedProjectId, this.selectedBillingAccountId)}">
+              CREATE PROJECT
+            </paper-button>
+          </span>
+            <div class="section">
+              <div class="section-header">
+                <span class="stepcircle">1</span>
+                <div class="instructions">
+                  Name your new Google Cloud Project
+                </div>
+              </div>
+              <div class="section-content">
+                <paper-input value="${
+        this.selectedProjectId}" label="Project ID" always-float-label="" maxlength="100" @value-changed="${
+        this.onProjectIdChanged}"></paper-input>
+              </div>
+            </div>
+            
+            <div class="section">
+              <div class="section-header">
+                <span class="stepcircle">2</span>
+                <div class="instructions">
+                  Choose your preferred billing method for this project
+                </div>
+              </div>
+              <div class="section-content">
+                <paper-dropdown-menu no-label-float="">
+                  <paper-listbox slot="dropdown-content" selected="${
+        this.selectedBillingAccountId}" attr-for-selected="name" @selected-changed="${
+        this.onBillingAccountSelected}">
+                  ${this.billingAccounts.map(billingAccount => {
+      return html`<paper-item name="${billingAccount.id}">${billingAccount.name}</paper-item>`;
+    })}
+                  </paper-listbox>
+                </paper-dropdown-menu>
+              </div>
+            </div>
+        </outline-step-view>
+
+        <outline-region-picker-step id="regionPicker" .localize=${
+        this.localize} @region-selected="${this.onRegionSelected}">  
+        </outline-region-picker-step>
+      </iron-pages>`;
+  }
+
+  private async handleBillingVerificationNextTap(): Promise<void> {
+    this.billingAccounts = await this.account.listBillingAccounts();
+    if (!this.billingAccounts || this.billingAccounts.length === 0) {
+      // TODO: Show error
+    } else {
+      this.showProjectSetup();
+    }
+  }
+
+  private isProjectSetupNextEnabled(projectId: string, billingAccountId: string): boolean {
+    // TODO: Proper validation
+    return projectId !== '' && billingAccountId !== '';
+  }
+
+  private async handleProjectSetupNextTap(): Promise<void> {
+    this.project =
```

Comment:
Need to handle failure. In case of error, display the error and open the project setup again (which should pick up on the new state).

Reply:
Yep, this is still a work in progress. Thanks for the suggestions.